Userland/Services/RequestServer/main.cpp

```diff
@@ -45,16 +45,11 @@ ErrorOr<int> serenity_main(Main::Arguments)
         TRY(Core::System::unveil("/home/anon", "rwc"));
     TRY(Core::System::unveil(nullptr, nullptr));
 
-    [[maybe_unused]] auto gemini = make<RequestServer::GeminiProtocol>();
-    [[maybe_unused]] auto http = make<RequestServer::HttpProtocol>();
-    [[maybe_unused]] auto https = make<RequestServer::HttpsProtocol>();
+    RequestServer::GeminiProtocol::install();
+    RequestServer::HttpProtocol::install();
+    RequestServer::HttpsProtocol::install();
 
     auto client = TRY(IPC::take_over_accepted_client_from_system_server<RequestServer::ConnectionFromClient>());
```

Comment:
An alternative would be to pass a vector of protocols to the ConnectionFromClient? and thus avoid the singleton map lookup. (or an object that represents a map of protocols, ProtocolSet, ProtocolManager, w/e)